packages/language-service/src/quick_info_built_ins.ts

```diff
@@ -137,80 +137,80 @@ const BUILT_IN_NAMES_TO_DOC_MAP: {
 } = {
   '@defer': {
     docString: `A type of block that can be used to defer load the JavaScript for components, directives and pipes used inside a component template.`,
-    links: ['[AIO Reference](https://next.angular.io/api/core/defer)'],
+    links: ['[AIO Reference](https://angular.dev/guide/defer#defer)'],
```

Comment:
```suggestion
    links: ['[Reference](https://angular.dev/guide/defer#defer)'],
```

nit: Remove AIO as well from the text

Reply:
I just did. I also made the links one bit more specific